awscli/examples/ecr-public/set-repository-policy.rst

```diff
@@ -0,0 +1,118 @@
+**Example1: To set repository policy to allow a pull on the repository**
```

Comment:
```suggestion
**Example 1: To set a repository policy to allow a pull on the repository**
```

The example title for doesn't seem to match the description ("pull on the repository" and "to control access permission").